python/eos_TEST.py

```diff
@@ -4,6 +4,9 @@
 import inspect
 import os
 
+import matplotlib.pyplot
+import matplotlib.texmanager
```

Comment:
hmm, I don't know why this is here. Probably due to some VScode stupid auto-include....